### PR TITLE
Add HPE documentation links to Grommet component screens

### LINKED ISSUE #397: Add link to HPE Design System docs to component pages
builds off of #402 

Right now, there is low awareness about how to transition from Grommet docs to HPE Design System guidance on a component. In order to raise visibility of HPE Design System docs and examples, a link should be added to each component page (alongside the Storybook, CodeSandbox, Github links) that leads to the Design System guidance for that component.

Deliverables:
- [ ] Adjust site layout to allow for "Design System Guidance" link when on HPE theme (tbd on label)
- [ ] Add links to each component page

### PR DESCRIPTION
**Pull request by JCoder121**
[Deploy Preview](https://deploy-preview-405--upbeat-ardinghelli-95952e.netlify.app/box)

image

image

Add Notification component allowing user to click a link redirecting to the HPE Design Site's documentation on the corresponding component for further reference. Not all Grommet components have dedicated documentation on DS (ex: Avatar, Sidebar, Color), so the notification will not appear. 
Closes #397.

### CODE REVIEW

File: src/components/Doc/ComponentDoc.js
Comment (taysea) [inline src/components/Doc/ComponentDoc.js:113]:
```suggestion
                label: `See ${name} examples and guidance.`,
```

File: src/components/Doc/ComponentDoc.js
Comment (taysea) [inline src/components/Doc/ComponentDoc.js:116]: @ericsoderberghp I know you aim to avoid inline styling as much as possible, but this feels like the cleanest to getting an underline under the text. Are you okay with this?

File: src/screens/Calendar.js
Comment (taysea) [inline src/screens/Calendar.js:46]: let's remove since we don't have a calendar page

File: src/components/Doc/ComponentDoc.js
Comment (ericsoderberghp) [reply src/components/Doc/ComponentDoc.js:116]: Something just feels existentially wrong to me about having inline styles in the Design System site. Can we just start with the default and see how it looks?

Comment (taysea) [reply src/components/Doc/ComponentDoc.js:116]: Yes, we can do that. @JCoder121 -- let's remove the inline style of underline and custom text color and just leave as the default grommet anchor treatment.

File: src/components/Doc/ComponentDoc.js
Comment (taysea) [inline src/components/Doc/ComponentDoc.js:114]: can we add `target: _blank` and `rel: noreferrer` so that it'll open in a new window?

File: src/components/Doc/ComponentDoc.js
Comment (taysea) [inline src/components/Doc/ComponentDoc.js:116]:
```suggestion
```
sorry -- I changed my mind, I think noreferrer could actually be helpful from DS analytics to know how many people are coming from grommet site.

**Review by taysea — APPROVED** [review]
LGTM!

File: src/components/Doc/ComponentDoc.js
Comment (ericsoderberghp) [inline src/components/Doc/ComponentDoc.js:91]: Why do we need to override the theme? Can we use the default?

Comment (JCoder121) [reply src/components/Doc/ComponentDoc.js:91]: I think it was because I couldn't find a way to make the default notification visually match what we agreed on, and it was easier to override the theme and manually specify certain parameters.

Comment (taysea) [reply src/components/Doc/ComponentDoc.js:91]: I think the main thing we wanted was to leverage Notification component for layout and text styling, but needed to add background color/rounding/direction that differs from grommet's theme.

https://storybook.grommet.io/?path=/story/visualizations-notification-status--status

Comment (taysea) [reply src/components/Doc/ComponentDoc.js:91]: Alternately, Jeffrey had explored just building the entire thing with Box, but getting to leverage Notification felt nice if possible (but does require theme overrides since these things aren't adjustable via props) @ericsoderberghp

Comment (taysea) [reply src/components/Doc/ComponentDoc.js:91]: Maybe the default grommet theming looks okay though and we just stick with that?

Comment (ericsoderberghp) [reply src/components/Doc/ComponentDoc.js:91]: Semantically, this feels like it should be a Notification, something to call attention to. If the grommet theme isn't working, then perhaps it means we should change the grommet theme. I'm inclined to use the components we've got as much as possible and only tweak them when we need to use them in distinct ways. All that said, I'll leave it up to the two of you to decide how you want to proceed. I'm OK taking this as is and submitting a new issue to re-evaluate the grommet theme for Notification, if preferred.

Comment (taysea) [reply src/components/Doc/ComponentDoc.js:91]: I chatted with Aditi and she didn't have any allergic reactions to the grommet default styling. In the spirit of keeping things lean, let's go with the default grommet styling for this first pass.

Next steps after merging -->

We can touch base with some of the developers I interviewed and open-ended let them know we made some small changes to the docs pages and let them browse. If they notice the new link, awesome! If no one notices it, that will indicate visibility is too low and we might need to rethink the visual treatment.

**Review by taysea — APPROVED** [review]
LGTM!

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**
